code/ship/ship.cpp

```diff
@@ -8856,10 +8856,67 @@ static void lethality_decay(ai_info *aip)
 #endif
 }
 
-void ship_process_pre(object *objp, float frametime)
+void ship_process_pre(object *obj, float frametime)
 {
-	if ( (objp == NULL) || !frametime )
+	// If Ai_before_physics is false everything following is evaluated in ship_process_post()
+	if ( (obj == nullptr) || !frametime || !Ai_before_physics)
 		return;
+
+	int	num;
+	ship* shipp;
+
+	if (obj->type != OBJ_SHIP) {
+		nprintf(("Network", "Ignoring non-ship object in ship_process_pre()\n"));
```

Comment:
There is probably a better category to put this under.  Network is for printing extra multiplayer statements.

Look around, there may be an AI or physics category?

Reply:
That makes sense for the -pre() version, since it's literally doing nothing but AI, but would about the -post() one? Is this something you would change for that as well?

Reply:
So after looking up the categories, imho "General" is probably the best choice here, because it doesn't look like there's anything more specific for physics or object types.